src/replica/replica_stub.cpp

```diff
@@ -790,6 +776,37 @@ void replica_stub::initialize(const replication_options &opts, bool clear /* = f
     }
 }
 
+void replica_stub::initialize_fs_manager(std::vector<std::string> &data_dirs,
```

Comment:
```const std::vector<std::string> &data_dirs```

Reply:
In function `initialize_fs_manager`, I should get the value of `data_dirs[i]` and `data_dirs_tag[i]`, using const cann't use index to get the value, so I remain it as original.